.github/workflows/cd_push.yml

```diff
@@ -1,5 +1,19 @@
 name: Workflow for building and deploying containers in argocd
 
+permissions:
```

Comment:
Can you add a comment that these are the default permissions with `id-token: write` added?